### ISSUE TITLE
Mathsteps cannot handle quadratic equations

### ISSUE THREAD

**Issue by Sakuya-Izayoi** [socraticorg/mathsteps#243]
When you pass a simple quadratic eqation like `2x^2 - 3x = 35` to the solveEquation function it just throws an error.
`Math error: No term with symbol: x, returning no steps`

**Comment by evykassirer** [comment]
yup, quadratics aren't super well supported in mathsteps right now

**Comment by mathematicode** [comment]
Hi there. I was hoping to contribute some code to solving quadratic equations for this project, but I was wondering what language it is all written in?

**Comment by arguiot** [comment]
@mathematicode if you can't recognize JavaScript, you'd better learn it before contributing 😂.

Also, I guess that solving quadratic equations shouldn't be too hard to implement, as you only have to transpose the equation to a polynomial, and then just apply the quadratic formula.

 ![Quadratic formula](https://latex.codecogs.com/svg.latex?%5Cfrac%7B-b%20%5Cpm%20%5Csqrt%7Bb%5E2%20-%204ac%7D%7D%7B2a%7D)

**Comment by kungfooman** [comment]
So any fork out there that supports quadratic equations by now?

Edit: Seems so: https://github.com/kemu-studio/mathsteps/blob/ce46d08aaffac60345bb9624275e21356739ed42/doc/20220426_forks_review.md?plain=1#L16
